TestUseEnum.test_assign_scoped_enum_value_name fails in 5.1.0

Trying to update the Fedora package to 5.1.0:
```
+ py.test-3.10 -v traitlets/tests/
============================= test session starts ==============================
platform linux -- Python 3.10.0rc2, pytest-6.2.4, py-1.10.0, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /builddir/build/BUILD/traitlets-5.1.0, configfile: pytest.ini
collecting ... collected 399 items
...
=================================== FAILURES ===================================
________________ TestUseEnum.test_assign_scoped_enum_value_name ________________
self = <traitlets.tests.test_traitlets_enum.TestUseEnum testMethod=test_assign_scoped_enum_value_name>
    def test_assign_scoped_enum_value_name(self):
        # -- CONVERT: string => Enum value (item)
        scoped_names = ["Color.red", "Color.green", "Color.blue", "Color.yellow"]
        for value in scoped_names:
            example = self.Example()
            example.color = value
            self.assertIsInstance(example.color, Color)
            if sys.version_info < (3, 10):
                self.assertEqual(str(example.color), value)
            else:
>               self.assertEqual(repr(example.color), value)
E               AssertionError: '<Color.red: 1>' != 'Color.red'
E               - <Color.red: 1>
E               ? -         ----
E               + Color.red
example    = <traitlets.tests.test_traitlets_enum.TestUseEnum.Example object at 0x7f774f58fe80>
scoped_names = ['Color.red', 'Color.green', 'Color.blue', 'Color.yellow']
self       = <traitlets.tests.test_traitlets_enum.TestUseEnum testMethod=test_assign_scoped_enum_value_name>
value      = 'Color.red'
traitlets/tests/test_traitlets_enum.py:94: AssertionError
```
I'm assuming this is related in some way to #661 and #663

--- a/traitlets/tests/test_traitlets_enum.py
+++ b/traitlets/tests/test_traitlets_enum.py
@@ -5,7 +5,6 @@ Test the trait-type ``UseEnum``.
 
 import unittest
 import enum
-import sys
 from traitlets import HasTraits, TraitError, Enum, UseEnum, CaselessStrEnum, FuzzyEnum
 
 
@@ -88,10 +87,7 @@ class TestUseEnum(unittest.TestCase):
             example = self.Example()
             example.color = value
             self.assertIsInstance(example.color, Color)
-            if sys.version_info < (3, 10):
-                self.assertEqual(str(example.color), value)
-            else:
-                self.assertEqual(repr(example.color), value)
+            self.assertEqual(str(example.color), value)
 
     def test_assign_bad_enum_value_name__raises_error(self):
         # -- CONVERT: string => Enum value (item)

